translations/hu.json

```diff
@@ -0,0 +1,2984 @@
+    }
+  },
+  "config guilds add-guild": {
+    "name": "guild-hozzáadás",
+    "description": "Hozzáad egy Hypixel guild-et a szerverhez.",
+    "options": [
+      {
+        "name": "guild",
+        "description": "A hozzáadandó guild."
+      }
+    ],
+    "responses": {
+      "addedGuild": [
+        "A **{guild}** nevű guild hozzá lett adva a szerverhez!"
+      ],
+      "guildNotFound": [
+        "A **{guild}** nevű guild nem létezik!"
+      ],
+      "premiumRequired": [
+        "Prémium szükséges ahhoz, hogy háromnál több guild-et adj hozzá a szerveredhez.\n",
+        "Több információért futtasd a /premium infó parancsot."
+      ],
+      "maxGuilds": [
+        "Nem tudsz 25-nél több guild-et hozzáadni."
+      ]
+    }
+  },
+  "config guilds remove-guild": {
+    "name": "guild-eltávolítás",
+    "description": "Eltávolít egy Hypixel guild-ot a szerverről",
+    "options": [
+      {
+        "name": "guild",
+        "description": "Az eltávolítandó guild."
+      }
+    ],
+    "responses": {
+      "removedGuild": [
+        "A **{guild}** nevű guild el lett távolítva a szerverről!"
+      ],
+      "guildNotFound": [
+        "A **{guild}** nevű guild nem létezik!"
+      ]
+    }
+  },
+  "config guilds edit-guild": {
+      "name": "guild-módosítás",
+      "description": "Módosítja a szerverhez kötött Hypixel guild konfigurációját.",
+      "options": [
+      {
+        "name": "guild",
+        "description": "A módosítandó guild."
+      },
+      {
+        "name": "lehetőségek",
+        "description": "A frissíthető lehetőségek.",
+        "choices": [
+          "Tagszámláló csatorna",
+          "Tagszámláló formátuma",
+          "Role"
+        ]
+      },
+      {
+        "name": "érték",
+        "description": "A beállítás új értéke. Hagyd üresen, ha alaphelyzetbe szeretnéd állítani."
+      }
+      ],
+      "responses": {
+        "guildNotFound": [
+          "A **{guild}** nevű guild does nem létezik."
+        ],
+        "editedGuild": [
+          "A **{guild}** nevű guild konfigurációja sikeresen frissítve lett."
+        ],
+        "premiumGuild": [
+          "Ez az opció csak a prémium szervereken használható.\n",
+          "Ha többet szeretnél tudni, használd a '/premium információ' parancsot."
+        ]
+      }
+  },
+  "config guilds add-rank": {
+    "name": "rang-hozzáadása",
+    "description": "Regisztrál egy guild rangot a bottal",
+    "options": [
+      {
+        "name": "guild",
+        "description": "A guild amelyiknél a rang lesz."
+      },
+      {
+        "name": "rang",
+        "description": "A regisztrálandó rang."
+      },
+      {
+        "name": "role",
+        "description": "A role ami a ranghoz lesz kötve. Hagyd üresen, ha nem akarsz role-t."
+      }
+    ],
+    "responses": {
+      "guildNotAdded": [
+        "A **{guild}** nevű guild nincs ehhez a szerverhez kötve.\n",
+        "Add a szerverhez a  `/config guilds guild-hozzáadás` paranccsal."
```

Comment:
The whole command name should be translated, including top level
Should also be in backticks for `this effect`